src/app/project/project-submission/project-submission.component.ts

```diff
@@ -9,6 +9,8 @@ import { ProjectService } from 'src/app/core/services/project.service';
 import { UserService } from 'src/app/core/services/user.service';
```

Comment:
This component is massive and should probably be evaluated for decomposition.  Additionally, formatting issues need to be fixed.

Reply:
Multiple console logs also being removed. There are multiple uses of alerts that should probably be replaced with a better notification to the client.